content/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -534,6 +530,7 @@ const WorkDetails: FunctionComponent<Props> = ({
                                   displayedContent={
                                     <p>
                                       <strong>(Title) </strong>
+                                      {/* Regex removes trailing full-stops.  */}
```

Comment:
Useful comment!

Thought for a separate patch:

We do something like this in a couple of places (search for the regex, you see it in three places). Maybe it would be worth pulling this into a separate named function in `grammar.ts` that adds a trailing full stop to a title? Rather than copying the regex (and this comment) around.